### ISSUE TITLE
Duplicate not-callable on property members

### ISSUE THREAD

**Issue by nelfin** [PyCQA/pylint#4426]
### Steps to reproduce

```python
# pylint: disable=missing-docstring,pointless-statement,invalid-name

class A:
    @property
    def value(self):
        return 42


a = A()
a.value()  # [not-callable]
```

### Current behavior

```
************* Module regression_duplicate_notcallable_with_property
tests/functional/r/regression/regression_duplicate_notcallable_with_property.py:10:0: E1102: a.value is not callable (not-callable)
tests/functional/r/regression/regression_duplicate_notcallable_with_property.py:10:0: E1102: a.value is not callable (not-callable)
```

### Expected behavior

Only a single instance of `not-callable` to be reported, e.g.

```
************* Module regression_duplicate_notcallable_with_property
tests/functional/r/regression/regression_duplicate_notcallable_with_property.py:10:0: E1102: a.value is not callable (not-callable)
```

### pylint --version output

Result of `pylint --version` output:

```
pylint 2.7.5.dev163+g841c51c
astroid 2.5.6
Python 3.9.0 (default, Nov 30 2020, 15:33:28) 
[GCC 5.4.0 20160609]
```
